acorn/src/parseutil.js

```diff
@@ -6,7 +6,7 @@ const pp = Parser.prototype
 
 // ## Parser utilities
 
-const literal = /^(?:'((?:\\.|[^'\\])*?)'|"((?:\\.|[^"\\])*?)")/
+const literal = /^(?:'((?:\\.|[^'\\])*?)'|"((?:\\.|[^"\\])*?)")/s
```

Comment:
```suggestion
const literal = /^(?:'((?:\\[\s\S]|[^'\\])*?)'|"((?:\\[\s\S]|[^"\\])*?)")/
```

Note that we still support `node>=0.4.0`

https://github.com/acornjs/acorn/blob/da521f6bcc7523d990ffb0ef61e364a7a22521ed/acorn/package.json#L20-L22

and the `s` flag is supported on Node.js 8.10+ only. 

Context: The Babel Node.js 6 test is breaking when using acorn 8.12: https://github.com/babel/babel/actions/runs/9767714418/job/26963669307?pr=16603